erpnext_com/www/docs/user/manual/en/CRM/lead.md

```diff
@@ -62,7 +62,7 @@ You can assign the Lead to a user by clicking on 'Assign' button on left bar. Yo
 
 ### 2.1 Reminders to Follow Up on the Leads
 
-It is important to reach out to leads from time to time and build the relationship. You can set the 'Next Contact Date' and 'Next Contact By' fields and a calendar event will be added for the user chosen in 'Next Contact By' field and a notification is shown on the that Date.
+It is important to reach out to leads from time to time and build the relationship. You can set the 'Next Contact Date' and 'Next Contact By' fields and a calendar event will be added for the user chosen in 'Next Contact By' field and a notification is shown on that Date.
```

Comment:
It is important to reach out to leads from time to time and build the relationship. For this, to contact the leads, you can set the 'Next Contact Date'. The 'Next Contact By' field allows you to set the user's name responsible to contact the lead. On setting this, an event will be created in the user's calendar and a notification will be sent for the same.